lib/makeup/lexers/erlang_lexer.ex

```diff
@@ -167,24 +167,12 @@ defmodule Makeup.Lexers.ErlangLexer do
   syntax_operators =
     word_from_list(~W[+ - +? ++ = == -- * / < > /= =:= =/= =< >= ==? <- ! ? ?!], :operator)
 
-  define =
-    token("define", :name_entity)
+  module_attribute =
+    token("-", :punctuation)
     |> optional(whitespace)
-    |> concat(token("(", :punctuation))
-    |> concat(token(macro_name, :name_constant))
-
-  record =
-    token("record", :name_entity)
+    |> concat(atom_name |> token(:name_attribute))
```

Comment:
I decided to be more permissive and generalized on what attributes/directives to match on here.
So we have no need for special matches for things like `module` or `record`.

But I'm open for discussion on this

Reply:
> I decided to be more permissive and generalized on what attributes/directives to match on here.

I agree with this